### DISCUSSION TITLE
Regarding showing system apps once all user apps are closed.

### DISCUSSION THREAD

**Discussion by visnkmr** [visnkmr/backgroundappslist#81] [Polls]
In the latest version of Background Apps and Process List we have made it such that the System apps list will be shown once all user apps are closed. We have some feedback against this change, Please do tell us which one you prefer:
If you want us to revert to previous behaviour of showing an empty list.

**Comment by franks06** [comment]
Please put it back to normal. This new way is too overwhelming

**Reply by lsilvestri2934** [reply]
I agree, unless there is a reason to see all the other files,

**Reply by franks06** [reply]
I have a customer who is in their 70s. This has cuasd them so much stress trying to get rid of 216 open things in the background. Please go back to the way it was ASAP. Thank you

**Comment by liberaxpress** [comment]
Put it back to how it was, some clients using this could think these are apps and start closing them, it's true!!
Maybe add another tab at the top for these if you think it is good to show them but don't have them on the same grouping as the standards app.

**Reply by franks06** [reply]
I had a customer unistall background apps 3 times and then he eventually whitelisted all the open apps to get them to close. It's not good, this way is making it very difficult to use and makes it seem lik all these are open in background. PLEASE GO BACK TO OLD WAY

**Comment by BuBByAU** [comment]
If the user wants the new behavior, let it be selected (not the default) under preferences menu.

**Comment by visnkmr** [comment]
We have updated the background apps list based on your feedback; should be available in 1-2 days. Now you can use the old layout and switch to system apps list with a button. We have also disabled the intro video autoplay. Please share your feedback on the update. We may remove the intro video entirely if that’s what you prefer.

**Reply by liberaxpress** [reply]
Another thing I noticed is when you accidentally click on one of the System apps list, if you click back you cannot come out and go back to the app home screen otherwise it will keep opening the rest of the apps 1 at a time unless you press the home button, then go to settings, manage installed applications and force stop on the Background app, this is certainly not good, glad you have decided to revert to original method...

**Comment by franks06** [comment]
Thank you. Can't wait for it go go back the way it was. This is too much going on with all these open apps.

Sent from Yahoo Mail on Android 
 
  On Tue, May 9, 2023 at 5:54 AM, ***@***.***> wrote:   


Another thing I noticed is when you accidentally click on one of the System apps list, if you click back you cannot come out and go back to the app home screen otherwise it will keep opening the rest of the apps 1 at a time unless you press the home button, then go to settings, manage installed applications and force stop on the Background app, this is certainly not good, glad you have decided to revert to original method...

**Comment by Forzic17** [comment]
![16910591674415669576722174243740](https://github.com/visnkmr/backgroundappslist/assets/141324579/8b006bd7-ad3a-4816-a43e-988a170ec6df)
Not working please

**Comment by visnkmr** [comment]
Hey,
From the version no it seems to be from Google Play but the version seems to be an old one. There should be an update already available on Google Play for the app. If this is being shown on a fire TV device, reinstalling the app from fire tvs appstore should fix the issue.

Thanks.
Vishnu N K
